### ISSUE TITLE
Sweeps for 1M word training set

### ISSUE THREAD

**Issue by MCooperBorkenhagen** [emelex-ai/BRIDGE#105]
Run sweeps for our 1M word training set, with the following hyperparameters:

d_model: 32, 64, 128
d_embedding: 1, 2
Number of phonology layers: 1, 2, 3, 4, 5, 6, 7, 8
Batch size: 32, 64
Learning rate: .001 and .0001
